src/main/app/fact/FactApi.ts

```diff
@@ -0,0 +1,20 @@
+import { Logger } from '../../types/Logger';
+import { AxiosInstance } from 'axios';
+
+export class FactApi {
+
+  constructor(
+    private readonly axios: AxiosInstance,
+    private readonly logger: Logger
+  ) { }
+
+  public search(query: string): Promise<unknown[]> {
```

Comment:
does this need to be here?

Reply:
I think it might become something like getAll - do we have an endpoint for that?